scripts/drone/versions/release.star

```diff
@@ -1,42 +1,59 @@
 load(
-    'scripts/drone/steps/lib.star',
```

Comment:
Given we've now got a `pipeline-steps` dir, could we go back to `pipelines`? I get where you're going with `versions`, i.e. versions of the build, but that term is overloaded (software versions).